.github/workflows/update-versions.yml

```diff
@@ -0,0 +1,66 @@
+name: Update Lotus Version
+
+on:
+  schedule:
+    - cron: '0 0 * * 1'  # Runs at 00:00 every Monday
+  workflow_dispatch:  # Allows manual trigger
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  update-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18'
+      
+      - name: Get latest Lotus version
+        id: get-version
+        run: |
+          # Get the latest release info
+          RELEASE_INFO=$(curl -s https://api.github.com/repos/filecoin-project/lotus/releases/latest)
+          
+          # Extract the tag name
+          TAG_NAME=$(echo "$RELEASE_INFO" | jq -r .tag_name)
+          
+          # Handle different tag formats
+          if [[ $TAG_NAME == *"miner"* ]]; then
+            # For miner tags, extract everything after 'v'
+            LATEST_VERSION=$(echo $TAG_NAME | sed -E 's/.*v([^/]*).*/\1/')
+          else
```

Comment:
Sorry, I realise I wasn't clear, we want to ignore the miner tag as it is technically a separate release stream for the lotus-miner software.